apps/examples/wc-next13/package.json

```diff
@@ -1,29 +1,32 @@
 {
   "name": "wc-next13",
-  "version": "0.3.43",
   "private": true,
+  "version": "0.3.43",
+  "type": "module",
   "scripts": {
-    "dev": "NODE_OPTIONS=--conditions=development next dev",
-    "build:examples": "yarn build-static",
-    "build-static": "next build && next export -o build",
-    "start": "next start",
-    "lint:examples": "next lint"
+    "dev": "vite",
+    "build": "vite build",
+    "lint": "eslint . --ext js,jsx --report-unused-disable-directives --max-warnings 0",
```

Comment:
`--report-unused-disable-directives` is very cool! Is it possible to add that to the config file?

Reply:
Sorry missed this comment. Do you mean in the lint config file?

Reply:
Sorry, yes, the eslintrc

Reply:
Seems to be part of eslint & CLI so i dont think this can be moved unfortunately.